chat-plugins/info.js

```diff
@@ -1821,10 +1822,10 @@ var commands = exports.commands = {
 			}
 			return this.sendReplyBox("Random number " + num + "x(1 - " + faces + "): " + rolls.join(", ") + "<br />Total: " + total);
 		}
-		if (target && isNaN(target) || target.length > 21) return this.sendReply("The max roll must be a number under 21 digits.");
```

Comment:
I might be wrong, but is it not possible to do `parseInt(parseFloat(target) 10)` to preserve scientific notation?